oneview_redfish_toolkit/blueprints/manager.py

```diff
@@ -41,41 +40,32 @@ def get_managers(uuid):
         /redfish/v1/Managers/id is requested.
 
         Returns:
-            JSON: JSON with Managers info for Enclosure or ServerHardware.
+            JSON: JSON with Managers info.
     """
     try:
-        appliance_information = \
+        oneview_appliances_version = \
```

Comment:
Since we have the uuid in the cache, why do we still need to retrieve from all OneViews?